persistence-modules/blaze-persistence/pom.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>blaze-persistence</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>blaze-persistence</name>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.4.0</version>
+        <relativePath/>
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <blaze-persistence.version>1.6.8</blaze-persistence.version>
+        <h2.version>1.4.200</h2.version>
+        <hibernate.version>5.4.12.Final</hibernate.version>
+        <junit-jupiter.version>5.7.0</junit-jupiter.version>
```

Comment:
You don't need to explicitly add this properties, you can used the inherited properties from parent modules.